command-line-interface.md

```diff
@@ -0,0 +1,189 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
+That executable MUST support commands with the following template:
+
+```sh
+$ funC [global-options] <COMMAND> [command-specific-options] <command-specific-arguments>
+```
+
+## Global options
+
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST NOT start with hyphens.
+The option parsing MUST be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` (even for commands not specified in this document).
+If the runtime is invoked with an unrecognized command, it MUST exit with a nonzero exit code and MAY log a warning to stderr.
+Beyond the above rules, the behavior of the runtime in the presence of commands and options not specified in this document is unspecified.
+
+## Character encodings
+
+This API specification does not cover character encodings, but runtimes SHOULD conform to their native operating system.
+For example, POSIX systems define [`LANG` and related environment variables][posix-lang] for [declaring][posix-locale-encoding] [locale-specific character encodings][posix-encoding], so a runtime in an `en_US.UTF-8` locale SHOULD write its [state](#state) to stdout in [UTF-8][].
+
+## Commands
+
+### create
+
+[Create][create] a container from a [bundle directory][bundle].
```

Comment:
please, no spec by example!  :-)

I would choose to not include each option in there just because it could get kind of long - but I'm not totally against it since we don't have too many.  Tested just a few linux commands and their usage line seemed to just show generic `[OPTIONS]` type of stuff - so I'd go for that. e.g. `Usage: tar [OPTION...] [FILE]...`